ui/selectors/selectors.js

```diff
@@ -97,9 +97,12 @@ export function getParticipateInMetaMetrics(state) {
 }
 
 export function isEIP1559Account(state) {
-  // Trezor does not support 1559 at this time
-  const currentKeyring = getCurrentKeyring(state);
-  return currentKeyring && currentKeyring.type !== KEYRING_TYPES.TREZOR;
+  const keyring = getCurrentKeyring(state);
+  if (keyring.type === KEYRING_TYPES.TREZOR) {
+    const model = keyring.getModel();
```

Comment:
This call isn't going to the TrezorKeyring handle; `getCurrentKeyring` in this instance is returning:

```
{
  "type": "Trezor Hardware",
  "accounts": [
    "0x9546aa6dfabfb3974db931096a4e6f84600cdb73"
  ]
}
```

So the `getModel` call is erroring out.

Reply:
I have addressed this in the latest commit 9a53bc270